napari/layers/base/base.py

```diff
@@ -1630,11 +1631,20 @@ def _display_bounding_box_augmented(
     ) -> npt.NDArray:
         """An augmented, axis-aligned (ndisplay, 2) bounding box.
 
-        This bounding box for includes the "full" size of the layer, including
-        for example the size of points or pixels.
+        This bounding box includes the size of the layer in best resolution, including required padding
         """
         return self._extent_data_augmented[:, dims_displayed].T
 
+    def _display_bounding_box_augmented_data_level(
+        self, dims_displayed: List[int]
+    ) -> npt.NDArray:
+        """An augmented, axis-aligned (ndisplay, 2) bounding box.
+
+        If the layer is multiscale layer, then returns the
+        bounding box of the data at the current level
+        """
+        return self._display_bounding_box_augmented(dims_displayed)
```

Comment:
I don't get how this is accounting for data level. Or is the doc string copy paste?

Reply:
Currently, only Image and Labels support multiscale data. So for all other layer types, the implementation will be identical to `_display_bounding_box_augmented_data`. But it is overwritten in `_ImageBase` to provide proper implementation for multiscale data.